packages/sdk/test/e2e/node/apiClient.spec.ts

```diff
@@ -43,4 +43,25 @@ describe("ApiClient Tests - Node", () => {
     assert.equal(res.data.url, "/foo");
     assert.equal(res.data.requestHeader!["authorization"], "Bearer test-bearer-token");
   });
+
+  it("can connect to existing API using bearer basic auth provider", async function () {
```

Comment:
remove `bearer` in the description to make it more accurate?

Reply:
Fixed.